distribution/docs/src/main/jdocs/content/_configuring/admin-console-tutorial-contents.adoc

```diff
@@ -0,0 +1,113 @@
+:title: Admin Console Tutorial
+:type: configuringAdminConsole
+:status: published
+:summary: Tutorial for the ${admin-console}.
+:order: 01
+
+.${admin-console} Tutorial
+****
+This overview covers general uses for the ${admin-console}.
```

Comment:
I wanted to use the different formatting to offset certain sections visually, but the pdf version doesn't handle it well, so I'll be removing it.